commands.json

```diff
@@ -167,7 +167,7 @@
     "group": "json"
   },
   "JSON.MERGE": {
-    "summary": "Sets or updates the JSON value at a path",
+    "summary": "Merge one or more values in a JSON document at `path` in `key`",
```

Comment:
Suggested:

Merges a given JSON value into matching paths. Consequently, JSON values at matching paths are updated, deleted, or expended with new children.

Reply:
Accepted in PR #1000